c_glib/test/dataset/test-file-system-dataset-factory.rb

```diff
@@ -109,9 +109,9 @@ def test_validate_fragments
                                     point: Arrow::Int16DataType.new)
       options.validate_fragments = true
       message = "[file-system-dataset-factory][finish]: " +
-                "Invalid: Unable to merge: " +
+                "Type Error: Unable to merge: " +
```

Comment:
```suggestion
                "Type error: Unable to merge: " +
```

Reply:
Thank you @kou I'm not very much into the ruby ecosystem or its casing :)

Reply:
Please ask me anything. :-)